
#try:
#     print(a)
# except Exception as e:
#     print(e)
# #else:
#     print("voila")
# print("hello")

try:
   num = int(input("Please enter a number: "))
   print(num)
except Exception as e:
   print("Invalid entry please try again")
